pkgs/top-level/all-packages.nix

```diff
@@ -29218,6 +29218,13 @@ with pkgs;
     libName = "librewolf";
   };
 
+  firedragon-unwrapped = callPackage ../applications/networking/browsers/firedragon {};
```

Comment:
To fix the CI you need to do exactly what it says, move to by-name as firedragon-unwrapped, then remove the above line from here. 

The firedragon override below should stay though.

Reply:
Well I was able to read that message 😋 but as stated in the PR I got an error doing it that way (`wasm-ld: error: unknown argument: --undefined-version`)

Reply:
> Well I was able to read that message 😋 but as stated in the PR I got an error doing it that way (`wasm-ld: error: unknown argument: --undefined-version`)

I'm pretty sure this is not related. Unless you have made your own (you must know) tricky patterns such as relative paths outside the import path.